sync-tool/UpdateFromPerlSrc.php

```diff
@@ -0,0 +1,189 @@
+<?php
+
+// http://cpansearch.perl.org/src/SBURKE/Text-Unidecode-1.23/lib/Text/Unidecode/
+
+use \Yaoi\String\StringValue;
+
+class UpdateFromPerlSrc extends Yaoi\Command
+{
+    public $libVer = '1.27';
+
+    static function setUpDefinition(\Yaoi\Command\Definition $definition, $options)
+    {
+        $options->libVer = \Yaoi\Command\Option::create()->setIsUnnamed()
+            ->setDescription('Version of Perl library');
+        $definition->name = 'update';
+        $definition->description = 'Tool for converting char tables for Behat/Transliterator from Perl to PHP';
+    }
+
+    public function performAction()
+    {
+        $rollingCurl = new \RollingCurl\RollingCurl();
```

Comment:
Please add use statements for classes instead of using FQCN in the code, to be consistent with our coding standards